xml/exits-xml.cpp

```diff
@@ -156,8 +167,16 @@ int Exit::saveToXml(xmlNodePtr parentNode) const {
     xml::saveNonNullString(rootNode, "Description", description);
     xml::saveNonNullString(rootNode, "Enter", enter);
     xml::saveNonNullString(rootNode, "Open", open);
-    effects.save(rootNode, "Effects");
 
+    std::list<std::string>::const_iterator ub;
+    if(!usedBy.empty()) {
+        childNode = xml::newStringChild(rootNode, "UsedBy");
+        for(ub = usedBy.begin() ; ub != usedBy.end() ; ub++) {
```

Comment:
nit: no need to change, but you can do `for(auto const& ply: usedBy)` since the structure won't be modified while iterating over it.